src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -166,6 +166,8 @@ case class WorkspaceListResponse(accessLevel: WorkspaceAccessLevel,
 case class WorkspacePermissionsPair(workspaceId: String,
                                     accessLevel: WorkspaceAccessLevel)
 
+case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, Boolean])
```

Comment:
we are losing error information here, maybe return Map[String, Option[ErrorReport]] or Map[String, Either[Boolean, ErrorReport]]